integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/PodUtils.java

```diff
@@ -439,4 +439,19 @@ public static Callable<Boolean> checkPodEvictedStatusInOperatorLogs(String opNam
       return isPodEvictedStatusLoggedInOperator(operatorLog, regex);
     };
   }
+
+  /**
+   * Check if the pod log contains the certain text.
+   * @param matchStr text to be searched in the log
+   * @param podName the name of the pod
+   * @param namespace namespace where pod exists
+   * @return true if the text exists in the log otherwise false
+   * @throws ApiException if Kubernetes client API call fails
+   */
+  public static boolean checkPodLogContains(String matchStr, String podName, String namespace)
+      throws ApiException {
+
+    return Kubernetes.getPodLog(podName,namespace,null).contains(matchStr);
```

Comment:
It may not matter, but just in case it wasn't considered, note that a long running operator could have 'matchStr' from a previous run of the test. This will make a future run of the test pass even when it doesn't generate the expected message. To mitigate, any log messages that were generated from _before_ the test is run should be ignored.

Reply:
you are right. In the test where I am using this method, there is only one test method in the class which causes the expected error message.